cli/npm/mod.rs

```diff
@@ -56,7 +55,74 @@ pub type CliNpmResolverCreateOptions =
 pub type CliByonmNpmResolverCreateOptions =
   ByonmNpmResolverCreateOptions<CliSys>;
 
-pub struct NpmPackageInfoApiAdapter(pub Arc<dyn NpmRegistryApi + Send + Sync>);
+pub struct NpmPackageInfoApiAdapter(
+  pub Arc<dyn NpmRegistryApi + Send + Sync>,
+  pub Arc<WorkspaceNpmPatchPackages>,
+);
```

Comment:
Nitpick: maybe use named properties at this point? (especially because it's public)